addons/mail/static/tests/message/message_tests.js

```diff
@@ -1302,8 +1369,9 @@ QUnit.test(
         openDiscuss(channelId);
         await contains(".o-mail-Message");
 
-        await click(".o-mail-AttachmentCard button[title='Remove']");
-        await click(".modal button", { text: "Ok" });
+        await click("[title='Expand']");
+        await click("[title='Delete']");
+        await click("button:contains(Confirm)");
```

Comment:
Make sure to rebase, and preferably use `await click("button", { text: "Confirm" })`.
This does the same but internally it doesn't use jQuery but native `querySelector()` instead.

_It's not that we don't like jQuery (for the syntax of selector, quite the contrary)... But we observed that very rarely `contains()` could make crash the CSS selector engine of jQuery (Sizzle), which results in some infinite loops. Because of that, unfortunately, we have to stay away from jQuery as much as we can, and come up with a custom function with params because the native version does not have the feature incorporated in CSS selector... And we definitely don't want to waste development effort to fix Sizzle and/or fork it for having a DSL with native browser features!_

Reply:
Hello @alexkuhn ,
Thanks for the review... I have applied the new changes. Can you please have a look?